src/config/environment.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose');
```

Comment:
Por que o file environment.js tem as mesmas coisas que o file localMongooseConnectionDB? Eu tinha perguntado se essa configuração de ambiente era geral ou só do mongo. Como é do mongo deixem em localMongooseConnectionDB. Não deixem código duplicado. Se ainda for existir o file environment.js coloquem config no final, no mesmo padrão dos outros files.

Reply:
Percebemos isso antes de olhar o PR e já retiramos, vamos commitar